sensor_msgs/Image image_2d
sensor_msgs/PointCloud2 point_cloud_3d
uint8 valid_depth
uint8 detection_active
uint8 image_flip
